t/react_template/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: 'Continuous Integration'
```

Comment:
Wondering should we add GA yamls, we can't garantee user is using github actions for CI. @drevell for opinion input. thanks

Reply:
As discussed in group, for now let's have them here. FE will be demoed in a separated repo

Reply:
Got it